EKF/mag_fusion.cpp

```diff
@@ -649,8 +649,10 @@ void Ekf::updateQuaternion(const float innovation, const float variance, const f
 		if (_control_status.flags.in_air) {
 			return;
 
-		} else {
+		} else if (isTimedOut(_time_last_in_air, (uint64_t)5e6)) {
 			// constrain the innovation to the maximum set by the gate
+			// we need to delay this forced fusion to avoid starting it
+			// immediately after touchdown, when the drone is still armed
```

Comment:
We need to return if not in air and not timed out

Reply:
Fixed with https://github.com/PX4/PX4-ECL/commit/d7459602c84b7faf615d9dd4ed5387a33aa34838